sound/soc/sof/sof-audio.h

```diff
@@ -118,6 +118,8 @@ struct snd_sof_dai {
 	const char *cpu_dai_name;
 
 	struct sof_ipc_comp_dai comp_dai;
+	int num_conf;
+	int def_conf;
```

Comment:
I prefer the longer names here as its easier to read ```number_configs``` and ```default_config```

Reply:
+1